lib/config/rules/react.js

```diff
@@ -92,6 +96,8 @@ module.exports = {
 
   // Enforce boolean attributes notation in JSX
   'react/jsx-boolean-value': 'error',
+  // Enforce or disallow spaces inside of curly braces in JSX attributes and expressions
+  'react/jsx-child-element-spacing': 'off',
```

Comment:
I think we’d want this rule on as well

Reply:
If Prettier already takes care of this for devs, I’m not sure this needs to be turned on

Reply:
I don't know if Prettier would take care of this? It's more of a clarification mechanism for developers, and in any case, Prettier is not necessarily being used when using this particular set of rules (if it does conflict with Prettier, we can turn it off just for the prettier config)

Reply:
Oh you're right, I was thinking of another rule that had to do with styles and has almost the same name. Turned on in 9978763